pkg/collector/corechecks/servicediscovery/module/impl_linux.go

```diff
@@ -124,8 +124,8 @@ func getSockets(p *process.Process) ([]uint64, error) {
 
 // namespaceInfo stores information related to each network namespace.
 type namespaceInfo struct {
-	// listeningSockets stores the socket inode numbers for sockets which are listening.
-	listeningSockets map[uint64]struct{}
+	// listeningSockets maps the socket inode numbers to port numbers for sockets which are listening.
+	listeningSockets map[uint64]uint16
```

Comment:
maybe we should have a struct as a value? it will allow us to make less changes if we will add other parameter (like proto)

Reply:
Done.